NebulaWorld/Factory/FactoryManager.cs

```diff
@@ -34,6 +35,30 @@ public static void Initialize()
             TargetPlanet = PLANET_NONE;
         }
 
+        public static void InitializePrebuildRequests()
+        {
+            //Load existing prebuilds to the dictionary so it will be ready to build
+            if (LocalPlayer.IsMasterClient)
+            {
+                using (GetPrebuildRequests(out var prebuildRequests))
+                {
+                    foreach (PlanetFactory factory in GameMain.data.factories)
+                    { 
+                        if (factory != null)
```

Comment:
Why are there nulls here?

Reply:
I am not sure, it was just "to be safe".